packages/components/src/InputTime/InputTime.tsx

```diff
@@ -52,3 +48,33 @@ export function InputTime({
     }
   }
 }
+
+function dateToTimeString(date?: Date): string {
+  if (date == undefined || !(date instanceof Date)) {
+    return "";
+  }
+
+  // Extract hours and minutes from the Date object
+  const hours = date.getHours().toString().padStart(2, "0");
```

Comment:
You could utilize [toTimeString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toTimeString) to do this in one line. If you think it is more readable to explicitly grab the hours and minutes then I am okay with that as well

Reply:
I think I prefer the reliability of the API over hoping the string format doesn't change and break the parsing.